python-sdk/example_dags/example_duckdb_load_transform_dataframe_and_save.py

```diff
@@ -0,0 +1,72 @@
+import os
+from datetime import datetime, timedelta
+
+import pandas as pd
+
+# Uses data from https://www.kaggle.com/c/shelter-animal-outcomes
+from airflow.decorators import dag
+
+from astro import sql as aql
+from astro.files import File
+from astro.table import Table
+
+DUCKDB_CONN_ID = "duckdb_conn"
+AWS_CONN_ID = "aws_conn"
+s3_bucket = os.getenv("S3_BUCKET", "s3://astro-sdk")
+
+
+@aql.transform()
+def filter_data(input_table: Table):
+    return """SELECT *
+    FROM {{input_table}} WHERE type NOT LIKE 'Guinea Pig'
+    """
+
+
+@aql.dataframe()
+def aggregate_data(df: pd.DataFrame):
+    new_df = df.pivot_table(index="DATE", values="NAME", columns=["TYPE"], aggfunc="count").reset_index()
+    new_df.columns = new_df.columns.str.lower()
+    return new_df
+
+
+@dag(
+    start_date=datetime(2021, 1, 1),
+    max_active_runs=1,
+    schedule_interval=None,
+    default_args={
```

Comment:
`retry_delay` 5min looks lots of time. I rather like to remove `default_args`.  but it completely your call

Reply:
I agree - we should make this as minimal as possible to highlight just what we're trying to show (duckdb transformations) and remove any non-required fields (default_args, max_active_runs)

Reply:
done https://github.com/astronomer/astro-sdk/pull/1695/commits/89a942d91af5885e387932194f289198b973506c